src/Exceptions/HttpHTTPVersionNotSupported.php

```diff
@@ -8,4 +8,3 @@ class HttpHTTPVersionNotSupported extends HttpException {
     public const DESCRIPTION = "Indicates that the server does not support, or refuses to support, the protocol version that was used in the request message.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/505";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces